src/Sylius/Bundle/ApiBundle/Form/EventSubscriber/AddUserFormSubscriber.php

```diff
@@ -9,7 +9,7 @@
  * file that was distributed with this source code.
  */
 
-namespace Sylius\Bundle\UserBundle\Form\EventSubscriber;
+namespace Sylius\Bundle\ApiBundle\Form\EventSubscriber;
```

Comment:
Is it truly a part of UserBundle? It not a subscriber which depends on how customer account work, which is a core think? 
Before we use this subscriber for Core CustomerType. After this fix this type has new subscriber, but the old one is still needed in ApiBundle.